packages/styles/scss/components/content-item-horizontal-media/_content-item-horizontal-media.scss

```diff
@@ -0,0 +1,210 @@
+    }
+  }
+  :host(#{$dds-prefix}-content-item-horizontal-copy) ::slotted(:not([slot])),
+  .#{$prefix}--content-item-horizontal__item--copy {
+    margin-bottom: $carbon--spacing-07;
+    @include carbon--breakpoint(sm) {
+      margin-bottom: $carbon--spacing-07;
+    }
+    @include carbon--breakpoint(md) {
+      margin-bottom: $layout-05;
+    }
+    @include carbon--breakpoint(lg) {
+      margin-bottom: $layout-05;
+    }
+    p {
+      color: $text-01;
+    }
+  }
+
+  :host(#{$dds-prefix}-content-item-horizontal-media)
+    .#{$prefix}--content-item__cta,
+  .#{$prefix}--content-item-horizontal__item--cta {
+    margin-top: auto;
+
+    .#{$prefix}--link-list {
+      padding: 0;
+      &:first-of-type {
+        padding: 0;
+      }
+    }
+
+    .#{$prefix}--link-list li {
+      @include carbon--breakpoint(md) {
+        margin-bottom: $carbon--spacing-05;
+      }
+    }
+  }
+
+  .#{$prefix}--content-item-horizontal__row {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+
+    @include carbon--breakpoint('sm') {
+      grid-template-columns: repeat(4, 1fr);
+      padding-left: 1rem;
+      padding-right: 1rem;
+    }
+
+    @include carbon--breakpoint('md') {
+      grid-template-columns: repeat(8, 1fr);
+      padding-left: 1rem;
+      padding-right: 1rem;
+    }
+
+    @include carbon--breakpoint('lg') {
+      grid-template-columns: repeat(16, 1fr);
+    }
+
+    @include carbon--breakpoint('max') {
+      padding-left: 1.5rem;
+      padding-right: 1.5rem;
+      margin-left: calc((100% - 99rem) / 2);
+      margin-right: calc((100% - 99rem) / 2);
+    }
+
+    .#{$prefix}--content-item-horizontal__col {
+      &:first-of-type {
+        @include carbon--breakpoint('sm') {
+          grid-column: 1 / span 4;
+        }
+
+        @include carbon--breakpoint('md') {
+          grid-column: 1 / span 6;
+          padding-right: $carbon--spacing-05;
+        }
+
+        @include carbon--breakpoint('lg') {
+          grid-column: 1 / span 11;
+          padding-right: $carbon--spacing-05;
+        }
+      }
+
+      &:last-of-type {
+        @include carbon--breakpoint('sm') {
+          grid-column: 1 / span 4;
+        }
+
+        @include carbon--breakpoint('md') {
+          grid-row: 1;
+          grid-column: 7 / span 2;
+          padding-left: $carbon--spacing-05;
+        }
+
+        @include carbon--breakpoint('lg') {
+          grid-row: 1;
+          grid-column: 12 / span 5;
+          padding-left: $carbon--spacing-05;
+        }
+      }
+    }
+
+    &.#{$prefix}--content-item-horizontal-media__align-right {
+      .#{$prefix}--content-item-horizontal__col {
+        &:first-of-type {
+          @include carbon--breakpoint('sm') {
+            grid-column: 1 / span 4;
+          }
+
+          @include carbon--breakpoint('md') {
+            grid-column: 3 / span 6;
+          }
+
+          @include carbon--breakpoint('lg') {
+            grid-column: 6 / span 11;
```

Comment:
Odd number of column span is rare for our Carbon grid usage, wanted to double-check if this was your intent.